docs/content/getting-started/docker.md

```diff
@@ -48,3 +48,17 @@ For example, for running cardano-node + ogmios on the testnet, listening to tcp/
 ```console
 $ NETWORK=testnet OGMIOS_PORT=1338 docker-compose --project-name cardano-ogmios-testnet up
 ```
+
+## As a single image
```

Comment:
As this is the default when building the image, maybe it's best to lead with this as it is certainly a faster way to get up and running.